test/syscalls/linux/kill.cc

```diff
@@ -102,6 +102,32 @@ TEST(KillTest, CanKillAllPIDs) {
   EXPECT_EQ(0, WEXITSTATUS(status));
 }
 
+TEST(KillTest, CannotTermReaper) {
+  int status;
+  EXPECT_THAT(kill(1, SIGTERM), SyscallSucceeds());
+  ASSERT_THAT(RetryEINTR(waitpid)(1, &status, 0),
+		      SyscallFailsWithErrno(ECHILD));
+}
+
+TEST(KillTest, CanTermNonReaper) {
+  pid_t pid = fork();
+  if (pid == 0) {
+    while (true) {
+      pause();
+    }
+  }
+
+  ASSERT_THAT(pid, SyscallSucceeds());
```

Comment:
Using "SyscallSucceeds" as a matcher here is a bit confusing, since the syscall is made on a different line.  Why not just ASSERT_GT(pid, 0) ?

Reply:
Thanks, fixed in new patch